helm/prometheus/README.md

```diff
@@ -0,0 +1,13 @@
+# Prometheus module
+Installs prometheus via a helm chart, this supports both gcp and aws
+
+## Usage
+
+```hcl
+module "prometheus" {
+  source         = "github.com/mozilla-it/terraform-modules//helm/prometheus?ref=master
+  enabled        = true
+  cloud_provider = "aws"
```

Comment:
this can't be auto-detected?

Reply:
No we can't or at least I am not aware of a way to auto detect this

Reply:
ok can put a pin in this one, not a big deal.